File: archivos1991/inject.js
const fs = require('fs');
const path = require('path');

const baseDir = "c:\\Users\\Admin\\Desktop\\Kuyén\\zonaArchivo";

let indexHtml = fs.readFileSync(path.join(baseDir, "index.html"), "utf-8");

const loreHtml = fs.readFileSync(path.join(baseDir, "pages", "lore.html"), "utf-8");
const facHtml = fs.readFileSync(path.join(baseDir, "pages", "factions.html"), "utf-8");
const entHtml = fs.readFileSync(path.join(baseDir, "pages", "entities.html"), "utf-8");
const monHtml = fs.readFileSync(path.join(baseDir, "pages", "monitor.html"), "utf-8");

indexHtml = indexHtml.replace("<!-- Lore -->", loreHtml);
indexHtml = indexHtml.replace("<!-- Factions -->", facHtml);
indexHtml = indexHtml.replace("<!-- Entities -->", entHtml);
indexHtml = indexHtml.replace("<!-- Monitor -->", monHtml);

fs.writeFileSync(path.join(baseDir, "index.html"), indexHtml, "utf-8");

console.log("Pages injected successfully into index.html via Node.js");

File: zonaArchivo/inject.py
import os

base_dir = r"c:\Users\Admin\Desktop\Kuyén\zonaArchivo"

with open(os.path.join(base_dir, "index.html"), "r", encoding="utf-8") as f:
    index_html = f.read()

with open(os.path.join(base_dir, "pages", "lore.html"), "r", encoding="utf-8") as f:
    lore_html = f.read()

with open(os.path.join(base_dir, "pages", "factions.html"), "r", encoding="utf-8") as f:
    fac_html = f.read()

with open(os.path.join(base_dir, "pages", "entities.html"), "r", encoding="utf-8") as f:
    ent_html = f.read()

with open(os.path.join(base_dir, "pages", "monitor.html"), "r", encoding="utf-8") as f:
    mon_html = f.read()

index_html = index_html.replace("<!-- Lore -->", lore_html)
index_html = index_html.replace("<!-- Factions -->", fac_html)
index_html = index_html.replace("<!-- Entities -->", ent_html)
index_html = index_html.replace("<!-- Monitor -->", mon_html)

with open(os.path.join(base_dir, "index.html"), "w", encoding="utf-8") as f:
    f.write(index_html)

print("Pages injected successfully into index.html")
